src/pages/products/sgid/water/nhd-lakes.astro

```diff
@@ -0,0 +1,110 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import DefinitionListItem from '@components/page/DefinitionListItem.astro';
+import ExternalLink from '@components/page/ExternalLink.astro';
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah Lakes NHD',
+  description: `National Hydrologic Dataset (NHD) lakes and other waterbodies.`,
+  stewards: ['UGRC', 'USGS'],
+  type: ProductType.POLYGON,
+  category: SgidCategory.WATER,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [`December 2016`],
+  hub: {
+    ...dataPages[metadata.title],
+  },
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      These data represent lake/pond features as well as other waterbodies like reservoirs, marshes, and playas. This
+      dataset is derived from the National Hydrology Database (NHD) compiled by the USGS in partnership with UGRC and
+      other states' GIS programs.
+    </p>
+  </Fragment>
+
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <Fragment slot="description">
+    <p>
+      UGRC has modified the source NHD data to create additional functionality. Two fields have been added to this
+      feature class (InUtah and IsMajor) and features have been split at the state boundary.
+    </p>
+    <p>
+      <dl>
+        <h3>NHD Codes</h3>
```

Comment:
**quibble**: headings should be above the `<dl>` and both elements should be outside of the `<p>` tag. Paragraphs should only contain text and links.